packages/backend/src/types.ts

```diff
@@ -51,6 +53,10 @@ export const mutedNoteReasons = ['word', 'manual', 'spam', 'other'] as const;
 export const followingVisibilities = ['public', 'followers', 'private'] as const;
 export const followersVisibilities = ['public', 'followers', 'private'] as const;
 
+export const exportableEntities = ['antenna', 'blocking', 'clip', 'customEmoji', 'favorite', 'following', 'muting', 'note', 'userList'] as const;
+
+export const importableEntities = ['antenna', 'blocking', 'customEmoji', 'following', 'muting', 'userList'] as const;
+
```

Comment:
これが何の値なのかの情報量が少ないように感じる

Reply:
```suggestion
export const exportableEntityKinds = ['antenna', 'blocking', 'clip', 'customEmoji', 'favorite', 'following', 'muting', 'note', 'userList'] as const;

export const importableEntitiyKinds = ['antenna', 'blocking', 'customEmoji', 'following', 'muting', 'userList'] as const;

```

とか…？ (何なのかわからない、というのは元からどうしようもなさそう)

Reply:
userExportableEntities / userImportableEntites とか？

Reply:
↑ + コメントとかで補足すれば良いんじゃないかしら

Reply:
```suggestion
// ユーザーがエクスポートできるエンティティ
export const userExportableEntities = ['antenna', 'blocking', 'clip', 'customEmoji', 'favorite', 'following', 'muting', 'note', 'userList'] as const;

// ユーザーがインポートできるエンティティ
export const userImportableEntities = ['antenna', 'blocking', 'customEmoji', 'following', 'muting', 'userList'] as const;

```

Reply:
エンティティというのが何を指しているのか明確にしたいかも

Reply:
> エンティティというのが何を指しているのか明確にしたいかも

良い感じの表現を募集中

Reply:
- この値は何に使われるのを想定しているのか？
- 既存のデータベースのテーブル名などと一致させる必要があるのか？それとも自由に決めていいものなのか？